android/maplibreui/src/main/java/com/stadiamaps/ferrostar/maplibreui/routeline/NavigationPathBuilder.kt

```diff
@@ -0,0 +1,23 @@
+package com.stadiamaps.ferrostar.maplibreui.routeline
+
+import androidx.compose.runtime.Composable
+import com.mapbox.mapboxsdk.geometry.LatLng
+import com.maplibre.compose.ramani.MapLibreComposable
+import com.stadiamaps.ferrostar.core.NavigationUiState
+
+data class NavigationPathBuilder(
```

Comment:
Thoughts on renaming to `PathOverlayBuilder`? I think this communicates what it does a bit better. We should probably also have a docstring too ;)

cc @Archdoog

Reply:
Discussed on the call today and @Archdoog came up with `RouteOverlayBuilder` to keep it consistent with how we talk about this elsewhere in the navigation stack.